app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreenController.kt

```diff
@@ -64,6 +64,7 @@ class ScanSimpleIdScreenController @Inject constructor(
 
     val keyboardEvents = Observable
```

Comment:
What do you feel about extracting the shared keyboard event stream outside this method and providing it as a parameter to this function? That way, we can also handle the hiding of the QR Code scanner view in a separate method instead of in this method.

Reply:
This is done.